ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -44,9 +46,17 @@ azure_connection::list_objects(std::vector<std::string> &objects) const
     return 0;
 }
 
+/*
+ * Puts an object given a reference to the filename
```

Comment:
Pls add a full stop at the end of your comment. Also, single line comments should be commented using // not

Reply:
address in https://github.com/wiredtiger/wiredtiger/pull/8718/commits/7685d04439a42c0c444b046c8fcd8744cb9ad9db